libs/sdk-ui-dashboard/src/_staging/layout/sizing.ts

```diff
@@ -352,3 +356,142 @@ export const getDashboardLayoutItemHeight = (size: IDashboardLayoutSize): number
 
 export const getDashboardLayoutItemHeightForGrid = (gridHeight: number): number =>
     gridHeight * GRID_ROW_HEIGHT_IN_PX;
+
+export const determineSizeForScreen = (
+    screen: ScreenSize,
+    layoutItemSize?: IDashboardLayoutSizeByScreenSize,
+): IDashboardLayoutSize => {
+    return {
+        ...(layoutItemSize ? layoutItemSize[screen] : {}),
```

Comment:
there could still be a null pointer exc when `layoutItemSize` does not have size for provided screen. This should handle it: 
`...(layoutItemSize ? layoutItemSize[screen] ?? {} : {}),`

Reply:
fixed